openfl/interface/interactive_api/experiment.py

```diff
@@ -233,19 +232,19 @@ def start(self, *, model_provider, task_keeper, data_loader,
     def define_task_assigner(self, task_keeper, rounds_to_train):
         """Define task assigner by registered tasks."""
         tasks = task_keeper.get_registered_tasks()
-        self.train_task_exist = False
+        train_task_exist = False
```

Comment:
Is there a reason to not define `train_task_exist `as `self.train_task_exist`

Reply:
This variable is used only inside this function. And this variables looks like crutches, I don't think that it makes sense to use them like class object attributes to provide them to user. But if need arises in the future, it's easy to fix.